### PR TITLE
chore(dev): add remote development script and docs

### LINKED ISSUE #3951: Add a run script to support the  new content.dev.lcip.org box
Basically need something like `FXA_ENV=remote npm start`

that would use config from here: https://github.com/mozilla/fxa-dev/pull/256#issuecomment-234091041

and will allow new contributors just install the content server and use remote servers for other stuff.

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes #3951

### CODE REVIEW

**Comment by vladikoff** [comment]
@vbudhram r?

**Comment by TDA** [comment]
LGTM! r+

File: README.md [hunk 1]
```diff
@@ -5,35 +5,27 @@
...
 
+## Quick Start
 
-Follow the instructions on:
-	[fxa-local-dev](https://github.com/mozilla/fxa-local-dev) to get a local version running.
-	
-The above link helps to install the following pre-requisites, and set up a development environment.
+Clone the repository, make sure you have [required prerequisites](https://github.com/mozilla/fxa-local-dev#dependencies) installed.
+Run `npm install` and `npm run start-remote`.
+This will start a local fxa-content-server on [http://127.0.0.1:3030](http://127.0.0.1:3030) that works with remote Firefox Accounts servers.
```
Comment (vbudhram) [inline README.md]: Is there any documentation that we can link to that shows how to configure `works with remote Firefox Accounts servers`? Or maybe add a command to open a new profile with correct server values?

**Comment by vbudhram** [comment]
Besides the one comment, LGTM 👍. I know that Firefox FxA config values have tripped me up before when testing.

File: README.md [hunk 2]
```diff
@@ -85,6 +77,7 @@ To change the default auth server edit `server/config/*.json` on your deployed i
 ## Servers
 
 * latest development - https://latest.dev.lcip.org/
+* content - https://latest.dev.lcip.org/
```
Comment (pdehaan) [inline README.md]: What's the difference between "latest development" and "content"? They both seem to be the same URL.

Comment (vladikoff) [inline README.md]: woops!

File: README.md (hunk 1 above)
Comment (vladikoff) [inline README.md]: Yeah good point, I'll open a follow up issue because we need to extract the Firefox launcher from fxa-local-dev first

**Pull request merged**